portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,20 +45,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    int count = 0;
-    ArrayList<Comment> comments = new ArrayList<Comment>();
+    ArrayList<Comment> comments = new ArrayList<Comment>(); 
 
-    for(Entity entity : results.asIterable()) {
+    // limit 10 comments
+    for(Entity entity : results.asIterable(FetchOptions.Builder.withLimit(10))) {
```

Comment:
I implemented Scott's comment here, does this work?

Reply:
Yes, I think it will work.